### ISSUE TITLE
App crashes when users adds new BEP20 token , then makes cold start

### ISSUE THREAD

**Issue by rafaelekol** [horizontalsystems/unstoppable-wallet-android#3245]
For example use this this contract address to add BEP20 token: 0x2170ed0880ac9a755fd29b2688956bd959f933f8

Crash log:

```
Process: io.horizontalsystems.bankwallet.dev, PID: 13094
    io.horizontalsystems.bankwallet.core.UnsupportedAccountException
        at io.horizontalsystems.bankwallet.core.managers.BinanceSmartChainKitManager.evmKit(BinanceSmartChainKitManager.kt:46)
        at io.horizontalsystems.bankwallet.core.factories.AdapterFactory.adapter(AdapterFactory.kt:46)
        at io.horizontalsystems.bankwallet.core.managers.AdapterManager.initAdapters(AdapterManager.kt:69)
        at io.horizontalsystems.bankwallet.core.managers.AdapterManager.access$initAdapters(AdapterManager.kt:19)
        at io.horizontalsystems.bankwallet.core.managers.AdapterManager$preloadAdapters$1.run(AdapterManager.kt:49)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.os.HandlerThread.run(HandlerThread.java:65)
```

**Comment by rafaelekol** [comment]
Caused by same coin id. This should be resolved after resolving similar coin codes issue.

**Comment by rafaelekol** [comment]
fixed by #3281
